app/enterprise/1.5.x/deployment/migrations.md

```diff
@@ -13,6 +13,7 @@ title: Migrating to 1.5
 * Kong Manager and the Admin API must share the same domain in order to use the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_cookies" target="_blank">SameSite</a> directive. If they are on separate domains, `cookie_samesite` must be set to `“off”`. Learn more in [Session Security](/enterprise/{{page.kong_version}}/kong-manager/authentication/sessions/#configuration-to-use-the-sessions-plugin-with-kong-manager)
 * Kong Manager must be served over HTTPS in order for the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies" target="_blank">Secure</a> directive to work. If using Kong Manager with only HTTP, e.g. on `localhost`, then `cookie_secure` must be set to `false`. Learn more in [Session Security](/enterprise/{{page.kong_version}}/kong-manager/authentication/sessions/#session-security)
 * The Kong Developer Portal has undergone a number of **Breaking Changes**, if you are currently using the Developer Portal, it will no longer work without manually migrating files, or enabling **Legacy Mode**. Learn more in [Kong Developer Portal - Whats new in 1.5](/enterprise/{{page.kong_version}}/developer-portal/overview)
+* The [Upstream TLS](https://docs.konghq.com/hub/kong-inc/upstream-tls/) plugin was deprecated in version 1.3, and removed in version 1.5. If you have configured this plugin, remove the plugin before upgrading to version 1.5. Otherwise, Kong Enterprise will fail to start due to the missing plugin.
```

Comment:
I think a relative rather than external absolute link would work here. Also we should start using the variables wherever we can.

```suggestion
* The [Upstream TLS](/hub/kong-inc/upstream-tls/) plugin was deprecated in version 1.3, and removed in version 1.5. If you have configured this plugin, remove the plugin before upgrading to version 1.5. Otherwise, {{site.ee_product_name}} will fail to start due to the missing plugin.
```